lib/sinon/extend.js

```diff
@@ -54,13 +52,12 @@ var hasDontEnumBug = (function () {
  *
  * Returns the extended target
  */
-function extend(target /*, sources */) {
+module.exports = function extend(target /*, sources */) {
     var sources = Array.prototype.slice.call(arguments, 1);
-    var source, i, prop;
-
-    for (i = 0; i < sources.length; i++) {
-        source = sources[i];
+    var i = 0;
+    var source, prop;
 
+    while ((source = sources[i++])) {
```

Comment:
This change doesn't actually change the behaviour of the code, but is is a departure from existing style. Please try to keep to the existing style.